recipes/newrelic/apm/java/linux.yml

```diff
@@ -184,434 +122,185 @@ install:
-
-          for systemdService in $(cat {{.TMP_DIR}}/systemd_restarted_services 2> /dev/null)
-          do
-            configFile=$(cat {{.TMP_DIR}}/systemd_$systemdService.configFile 2> /dev/null)
-
-            systemctl status $systemdService > /dev/null || echo -e "
-              ${red}The systemd service \"$systemdService\" did not restart properly.
-              Revert by restoring the backup of $configFile
-              ${noColor}"
-          done
-
-          for supervisordService in $(cat {{.TMP_DIR}}/supervisord_restarted_services 2> /dev/null)
-          do
-            configFile=$(cat {{.TMP_DIR}}/supervisord_$supervisordService.configFile 2> /dev/null)
-
-            supervisorctl status $supervisordService | grep " RUNNING " > /dev/null || echo -e "
-              ${red}The supervisor service \"$supervisordService\" did not restart properly.
-              Revert by restoring the backup of $configFile
-              ${noColor}"
-          done
+          (exit 0)
+          EOT
+          sudo sed -i "s/DEFAULT_APP/$defaultAppName/g" /etc/newrelic-java/dynamic-attach.sh
+          sudo sed -i "s/RENAME_DOCKER/$renameDocker/g" /etc/newrelic-java/dynamic-attach.sh
+          sudo chmod 700 /etc/newrelic-java/dynamic-attach.sh
 
-    notify-done:
+    add-on-host-integration:
       cmds:
         - |
-          YELLOW='\033[0;33m'
-          NOCOLOR='\033[0m'
-          echo -e "${YELLOW}
-          ================================================================================
-          =                                                                              =
-          =                                  Important                                   =
-          =                                                                              =
-          =              Your application is sending JVM data to New Relic.              =
-          =                                                                              =
-          =      To view transaction data, send a few requests to your application.      =
-          =                                                                              =
-          ================================================================================
-          ${NOCOLOR}
-          "
-
-    cleanup:
-      cmds:
-        - |
-          rm -rf {{.TMP_DIR}}
-          isYumInstalled=$(which yum 2>&1 || true)
-          if [[ -n "$isYumInstalled" ]]; then
-            yum remove -y -q nri-introspector-java &> /dev/null
-          else
-            apt remove -y -qq nri-introspector-java &> /dev/null
+          if [ -f /etc/newrelic-infra/integrations.d/java-dynamic-attach-config.yml ]; then
+            rm -f /etc/newrelic-infra/integrations.d/java-dynamic-attach-config.yml
           fi
-
-
+          tee -a /etc/newrelic-infra/integrations.d/java-dynamic-attach-config.yml > /dev/null <<"EOT"
+          integrations:
+            - name: dynamic-attach
```

Comment:
@mbruzina maybe name java-dynamic-attach

Reply:
I'll clean that up. For some reason I thought *config.yml was a convention for OHIs.

Reply:
@mbruzina FYI I think this has been renamed above in the file for build_stdout_json( ) but not for the integration config file

Reply:
I don't see any references in this branch with *-config.yml 🤔